symbology/custom-dictionary-style/README.md

```diff
@@ -0,0 +1,42 @@
+# Custom dictionary style
+
+Use a custom dictionary style (.stylx) to symbolize features using a variety of attribute values.
+
+![Custom dictionary style](CustomDictionaryStyle.png)
+
+## Use case
```

Comment:
I think the customizable apsect is adressed with 'you can build each symbol on-the-fly', but am happy to take any suggestions on how to make it clearer?

Reply:
re-reading it, it's probably clear enough from the first sentence intro to the readme. Just not super clear in the use case, since it's not the dictionary renderer that is customizable but the custom dictionary style itself.